tests/Integration/Behaviour/Features/Context/Domain/Product/SpecificPriceContext.php

```diff
@@ -177,6 +178,22 @@ public function editSpecificPrice(string $specificPriceReference, TableNode $tab
         }
     }
 
+    /**
+     * @When I delete specific price ":specificPriceReference"
+     *
+     * @param string $specificPriceReference
+     */
+    public function deleteSpecificPrice(string $specificPriceReference): void
+    {
+        $specificPriceId = $this->getSharedStorage()->get($specificPriceReference);
+        try {
+            $command = $this->createDeleteSpecificPriceCommand($specificPriceId);
+            $this->getCommandBus()->handle($command);
```

Comment:
Considering how small the method is it may be overkill

```suggestion
            $this->getCommandBus()->handle(new DeleteProductSpecificPriceCommand($specificPriceId));
```